index.html

```diff
@@ -60,5 +60,172 @@ <h2>I’m Ahmed Glad to see you!</h2>
           
 
     </section>
+ 
+    <section class="works">
+      <div class="card-1">
+        <img src="assets/images/Snapshoot Portfolio-1.png" alt="Snapshoot-portfolio">
+        <div class="info-block">
+          <div class="client">
+            <h2 class="project-title-1">Tonic</h2>
+            <div class="details-1">
+              <div class="name"><p class="first-project-name">CANOPY</p></div>
+              <div class="curcle-1"></div>
+              <div class="role"><p class="first-project-role">Back End Dev</p></div>
+              <div class="curcle-2"></div>
+              <div class="year"><p class="first-project-year">2015</p></div>
+            </div>
+          </div>
+          <p class="summary-1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="languages-1">
+            <div class="html"><p class="first-project-html">html</p></div>
+            <div class="css"><p class="first-project-css">css</p></div>
+            <div class="js"><p class="first-project-js">javaScript</p></div>
+          </div>
+          <div class="action">
+           <button><p class="first-button">See Project</p></button>
+          </div>
```

Comment:
- [ ] :thinking: In order to repeat the HTML semantic rules and SEO, I think it would be better to remove the p tag inside the button :recycle: